ThunderCloud/UINavigationController+TSCNavigationController.m

```diff
@@ -225,9 +225,20 @@ - (void)TSC_handleWeb:(TSCLink *)link
         NSOperatingSystemVersion iOS9 = (NSOperatingSystemVersion){9, 0, 0};
         if ([[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:iOS9]) {
             
-            SFSafariViewController *safariViewController = [[SFSafariViewController alloc] initWithURL:link.url];
-            safariViewController.delegate = self;
-            [self presentViewController:safariViewController animated:true completion:nil];
+            NSURL *url;;
+            
+            if ([[link.url.absoluteString substringToIndex:7] isEqualToString:@"http://"] || [[link.url.absoluteString substringToIndex:8] isEqualToString:@"https://"]) {
```

Comment:
Isn't this going to throw an exception if the URL is nil? we get a lot of empty links from the CMS.